src/commands/config.js

```diff
@@ -0,0 +1,82 @@
+// Import Node.js Dependencies
+import os from "node:os";
+import path from "node:path";
+import fs from "node:fs/promises";
+
+// Import Third-party Dependencies
+import * as RC from "@nodesecure/rc";
+import kleur from "kleur";
+import { spawn } from "node:child_process";
+
+const K_HOME_PATH = path.join(os.homedir(), "nodesecure");
+
+async function spawnCodeAtNodeSecureRc(path, isCwd = false) {
+  try {
+    await fs.access(path);
+    spawn("code", [path]);
```

Comment:
Not sure if dealing with Asynchronous API is required here (as we are in a CLI).